### PR TITLE
Fixing qp_query_execution tests for gporca [#117472633].

### PR DESCRIPTION
**Pull request by foyzur**
Introduced separate plan support for optimizer vs. planner in qp_query_execution tests. Fixed the sql and answer files to make the qp_query_execution test green.

### CODE REVIEW

File: src/test/regress/expected/qp_query_execution.out [hunk 1]
```diff
@@ -176,10 +176,10 @@ insert into bar select i % 7, i % 6, i % 9, i || 'SOME NUMBER', i % 4 from gener
 analyze foo_p;
 analyze bar;
 -- start_ignore
-select qx_count_operator('explain select foo_p.b, foo_p.t from foo_p left outer join bar on foo_p.a = bar.k  where foo_p.t is not null and foo_p.a = 6;', 'Hash Left Join');
+select qx_count_operator('explain select foo_p.b, foo_p.t from foo_p left outer join bar on foo_p.a = bar.k  where foo_p.t is not null and foo_p.a = 6;', 'Hash Left Join', 'Hash Left Join');
  qx_count_operator 
 -------------------
-                 1
+                 0
 (1 row)
```
Comment (vraghavan78) [inline src/test/regress/expected/qp_query_execution.out]: is this supposed to be 1?

May be we can remove Ignore, and request NLJ for planner?

**Comment by vraghavan78** [comment]
Thanks @foyzur

**Comment by danielgustafsson** [comment]
Thanks for the fix, confirmed to build green in the buildfarm.

**Comment by foyzur** [comment]
Thanks everyone! Closing :-)

**Comment by foyzur** [comment]
Pushed to master: 4d7a3fe1c2f0f4d7361b1107a7267f318a122a7a

**Pull request closed without merging**